src/storage/src/hummock/sstable_store.rs

```diff
@@ -869,14 +869,22 @@ mod tests {
     async fn validate_sst(
```

Comment:
`validate_sst` only used for test, how about add cfg(test) and rename it to `validate_sst_for_test` ?

Reply:
No, it is already  in test mod ....